apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -346,6 +367,23 @@ function LandingPage({
   const RenderMyPlTab = () => {
     return (
       <>
+        {showEnrollmentSuccessMessage && (
```

Comment:
It looks like in Figma that this will be replaced with the dialog that Hannah is building. Is that right?

Reply:
Yep! I have a followup ticket for it to implement it once both hers and mine are merged!